src/hssm/distribution_utils/dist.py

```diff
@@ -117,13 +117,16 @@ class SSMRandomVariable(RandomVariable):
 
         # NOTE: This is wrong at the moment, but necessary
         # to get around the support checking in PyMC that would result in error
-        ndim_supp: int = 0
+        ndim_supp: int = 1
 
         ndims_params: list[int] = [0 for _ in list_params]
         dtype: str = "floatX"
         _print_name: tuple[str, str] = ("SSM", "\\operatorname{SSM}")
         _list_params = list_params
 
+        def _supp_shape_from_params(*args, **kwargs):
```

Comment:
can you add a comment above this to explain (for future reference) what this does, and link to issue?